RecoTracker/TkSeedGenerator/plugins/DeepCoreSeedGenerator.cc

```diff
@@ -560,8 +561,8 @@ void DeepCoreSeedGenerator::fillDescriptions(edm::ConfigurationDescriptions& des
   desc.add<edm::InputTag>("vertices", edm::InputTag("offlinePrimaryVertices"));
   desc.add<edm::InputTag>("pixelClusters", edm::InputTag("siPixelClustersPreSplitting"));
   desc.add<edm::InputTag>("cores", edm::InputTag("jetsForCoreTracking"));
-  desc.add<double>("ptMin", 300);
-  desc.add<double>("deltaR", 0.1);
+  desc.add<double>("ptMin", 100);
+  desc.add<double>("deltaR", 0.25);  // the current training makes use of 0.1
```

Comment:
the PR description has 
> 3. build seeds via deepCore in a cone of at least 0.2 wrt the jet axis (instead of the current 0.1)

is the code here supposed to change or is it the description that's out of date?

Reply:
> the PR description has
> 
> > 1. build seeds via deepCore in a cone of at least 0.2 wrt the jet axis (instead of the current 0.1)
> 
> is the code here supposed to change or is it the description that's out of date?

please comment

Reply:
I think I updated the PR description